src/marqo/tensor_search/tensor_search.py

```diff
@@ -558,6 +562,7 @@ def _add_documents_structured(config: Config, add_docs_params: AddDocsParams, ma
     total_vectorise_time = 0
     batch_size = len(add_docs_params.docs)  # use length before deduplication
     image_repo = {}
+    text_chunk_prefix = determine_text_prefix(add_docs_params.text_chunk_prefix, marqo_index, "text_chunk_prefix")
```

Comment:
It's redundant for the string to be "text_chunk_prefix", what if it's just "text_chunk" or "text_query".

Reply:
We're only getting the prefix here, which we then pass on to various functions in different parts of the function. so setting it to `text_chunk` or `text_query` would make its use unclear. Perhaps `text_prefix` would be better?

Reply:
Actually `text_chunk_prefix` would be good too since we don't expect `text_query_prefix` here in the add_documents function

Reply:
Ah that's not what I meant, the variable name is good as `text_chunk_prefix`. I meant the 3rd argument for the `determine_text_prefix` function.